src/Sprites/riverbank.py

```diff
@@ -0,0 +1,33 @@
+import pygame.sprite
+
+
+class Riverbank(pygame.sprite.Sprite):
+    """Pygame sprite class for frog nests used for checking the win condition"""
```

Comment:
Document ya functions bro, this is copied from the nest class 😂

Reply:
heh...heh....guilty as charged xD